src/com/bogdan/shop/ShopConfiguration.java

```diff
@@ -21,11 +20,11 @@ public Money getMoney() {
         return money;
     }
 
-    public ProductList getProductsList() {
-        if (productsList == null) {
-            productsList = new ProductList(getMoney());
+    public ProductList getProductList() {
```

Comment:
return list of products instead of ProductList(and pls delete this class)

Reply:
use this for initialization of products -> List.of(new Products(), new Products())